functests/execsboms.py

```diff
@@ -280,19 +280,22 @@ def test_sbom_upload_and_download(self):
             print(i, ":", json_dumps(m.dict(), indent=4))
 
 
-class TestSBOMWithApplication(TestSBOM):
-    """
-    Test Archivist SBOM upload/download
-    """
-
-    maxDiff = None
-
-    def setUp(self):
-        super().setUp()
-        self.title = "TestSBOMWithApplication"
-        application = self.arch.applications.create(
-            DISPLAY_NAME,
-            CUSTOM_CLAIMS,
-        )
-        auth = (application["client_id"], application["credentials"][0]["secret"])
-        self.arch = Archivist(getenv("TEST_ARCHIVIST"), auth, verify=False)
+# class TestSBOMWithApplication(TestSBOM):
```

Comment:
Is this meant to be commented out?

Reply:
yes - its deadcode and I was testing if it was needed somewher and its not so deleted it